ReadMe.md

```diff
@@ -46,11 +48,18 @@ This will install the following dependencies:
 - opencv-python == 4.8.0.74
 - numpy == 1.24.3
 
- Step 3: Running the Backend
+ Step 3: After installing the dependencies, navigate to the backend directory, create a .env file in the backend directory:
 
-After installing the dependencies, navigate to the backend directory and run the Flask application:
+    cd backend
+
+    After creating .env file, add these lines in it:
+
+    FLASK_RUN_HOST=127.0.0.1
+    FLASK_RUN_PORT=5000
```

Comment:
So we don't want to create a personal configuration file every time someone sets up a local version of our code.

The goal of these environment files is to separate configuration from our source code.

These environment variables should dynamically configure these values depending on the environment their being executed in so we dont want to gitignore them either. So for example, endpoints for uploading, server configs etc can be stored & accessed in these types of configuration files & loaded by the code.

Reply:
This means we should have reasonable defaults that automatically configure dev environments as well. 

If a port isn't open or a dev wants to change anything else they can do that but there should be existing dev configurations that already exist for them to modify

Reply:
so, If I remove the env file from the .gitignore, it will help us achieve that, right? There will be a default env file for the developer to change if needed.